lang/csharp/src/apache/codegen/Avro.codegen.csproj

```diff
@@ -55,15 +55,6 @@
     <WarningsAsErrors />
   </PropertyGroup>
 
-  <!-- See lang/csharp/README.md for tool and library dependency update strategy -->
-  <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
-    <PackageReference Include="System.CodeDom" Version="$(SystemCodeDomVersion)" />
-    <PackageReference Include="System.Reflection" Version="$(SystemReflectionVersion)" />
-    <PackageReference Include="System.Reflection.Emit.ILGeneration" Version="$(SystemReflectionEmitILGenerationVersion)" />
-    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="$(SystemReflectionEmitLightweightVersion)" />
-  </ItemGroup>
```

Comment:
I am not C# developer but this change does not look related to the issue with the namespaces by avrogen.
Please explain.

Reply:
Yes. It is not related. I just got a watning that those packages are not used. All the reflection related code is in the main library. I guess they used to be code in avrogen which used relection, however it is not present.  I can move it to a seperate ticket easily if needed. I considered this change just like removing unused `using WhateverPackage;`

Reply:
Btw when I do sneaky stuff like this, usually create a  specific commit for it and explain somewaht in the commit mnessage. https://github.com/apache/avro/pull/1610/commits/6326ee3d262c1e77957c096ba611dabd88c8a8b5